salt/metalk8s/addons/ui/files/metalk8s-ui-deployment.yaml

```diff
@@ -52,7 +57,13 @@ spec:
           - name: metalk8s-ui-volume
             mountPath: /etc/metalk8s/ui
             readOnly: true
+          - name: ui-branding-volume
+            mountPath: /etc/metalk8s/ui/brand
```

Comment:
Not sure about this, what about order and risk of overwrite?

Reply:
I have no idea. It seems to work well. To make it more robust, we should mount ```metalk8s-ui-volume``` in ```/etc/metalk8s/ui/config``` instead of ```/etc/metalk8s/ui```?